packages/spec-test-util/src/single.ts

```diff
@@ -112,7 +112,7 @@ export function describeDirectorySpecTest<TestCase extends {meta?: any}, Result>
         continue;
       }
 
-      // Use full path here, not just `testSubDirname` to allow usage of `vitest --grep`
+      // Use full path here, not just `testSubDirname` to allow usage of `vitest -t`
```

Comment:
completely unrelated change, picked that up while reviewing https://github.com/ChainSafe/lodestar/pull/6222 and since this is a fixup of that PR, might as well include that here